demos/index.html

```diff
@@ -8,37 +8,38 @@
         <link rel="stylesheet" type="text/css" href="../src/lib/infusion/src/components/overviewPanel/css/OverviewPanel.css" />
         <link rel="stylesheet" type="text/css" href="../src/lib/infusion/src/components/inlineEdit/css/InlineEdit.css" />
 
-        <script type="text/javascript" src="../src/lib/infusion/infusion-custom.js"></script>
-        <script type="text/javascript" src="../src/lib/d3/d3.js"></script>
-
-        <script type="text/javascript" src="../src/lib/flocking/flocking-base.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/oscillators.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/math.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/envelopes.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/midi.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/scheduling.js"></script>
-
-        <script type="text/javascript" src="../src/js/valueBinding.js"></script>
-        <script type="text/javascript" src="../src/js/percentage.js"></script>
-        <script type="text/javascript" src="../src/js/transformations.js"></script>
-        <script type="text/javascript" src="../src/js/templateInjection.js"></script>
-        <script type="text/javascript" src="../src/js/d3ViewComponent.js"></script>
-        <script type="text/javascript" src="../src/js/totalRelaying.js"></script>
-        <script type="text/javascript" src="../src/js/legend.js"></script>
-        <script type="text/javascript" src="../src/js/pie.js"></script>
-        <script type="text/javascript" src="../src/js/pieChart.js"></script>
-        <script type="text/javascript" src="../src/js/d3Utils.js"></script>
-        <script type="text/javascript" src="../src/js/utils.js"></script>
-        <script type="text/javascript" src="../src/js/dataEntry.js"></script>
-        <script type="text/javascript" src="../src/js/dataEntryPanel.js"></script>
-        <script type="text/javascript" src="../src/js/electricPianoBand.js"></script>
-        <script type="text/javascript" src="../src/js/unitDivisorSonificationStrategy.js"></script>
-        <script type="text/javascript"
-        <script type="text/javascript" src="../src/js/sonifier.js"></script>
-        <script type="text/javascript" src="../src/js/chartAuthoring.js"></script>
-
-        <script type="text/javascript" src="src/js/chartAuthoring-demo-overview.js"></script>
-        <script type="text/javascript" src="src/js/chartAuthoring-demo.js"></script>
+        <script  src="../src/lib/infusion/infusion-custom.js"></script>
```

Comment:
All of Flocking's unit tests pass against the latest Infusion dev release. @waharnum and I discussed this in the channel and it looks like an older version of Flocking was lingering around due to the built files not being updated after an <code>npm install</code>.

I have, for the record, updated Flocking's master branch with this latest Infusion release.